### DISCUSSION TITLE
Django web app vs Quasar

### DISCUSSION THREAD

**Discussion by rkmullapudi** [quasarframework/quasar#15497] [CLI - SSR mode]
Hi, 

We have a web application running with the following stack:

Django
Python
PostgreSQL
Linux hosting
Vue Js/Javascript

We would like to know whether we can use Quasar to make our web application compatible for all the devices?

Best regards,
~RK

**Comment by metalsadman** [comment]
Yeah why not, quasar is frontend you can use it with any backend.

**Comment by zvolsky** [comment]
Can somebody write more?
I am sure about development configuration, on localhost Django will run at :8000 (runserver) and Quasar at :9000 (quasar dev).

I know I can then run `quasar build` and inside the produced directory dist/spa/ I can run `quasar serve`, Quasar is now at :4000.

But how it should be arranged in real production?
Plain Django uses lets say nginx+gunicorn+django.
Lets say Django and frontend both works on some domain `www.example.com`, ie. nginx should separate urls for Quasar and for Rest (gunicorn + Django/DRF)?
Should `quasar serve` be used, or something else?
